docs: update documentation for Cypress reporter (#551)
qase-tms/qase-javascript: qase-javascript-commons/README.md

# Qase JavaScript Commons

This package contains common classes and functions for working with Qase TMS API.

## Installation

```bash
npm install qase-javascript-commons@beta
```

## Configuration

Qase JS Reporters can be configured in multiple ways:

- using a config file `qase.config.json`
- using environment variables

All configuration options are listed in the table below:

> 

Reviewer: I hope you know about https://www.tablesgenerator.com/markdown_tables

Reviewer: It makes editing markdown tables much easier

Author: I didn't know about this site. Thanks!

| Driver used for report mode                                                                                                                  | `report.driver`            | `QASE_REPORT_DRIVER`            | `local`                                 | No       | `local`                    |
| Path to save the report                                                                                                                      | `report.connection.path`   | `QASE_REPORT_CONNECTION_PATH`   | `./build/qase-report`                   |          |                            |
| Local report format                                                                                                                          | `report.connection.format` | `QASE_REPORT_CONNECTION_FORMAT` | `json`                                  |          | `json`, `jsonp`            |
| **Qase TestOps configuration**                                                                                                               |                            |                                 |                                         |          |                            |

> | Token for API access, you can find more information [here](https://developers.qase.io/#authentication)                                       | `testops.api.token`        | `QASE_TESTOPS_API_TOKEN`        |                                         | Yes      | Any string                 |

Reviewer: ```suggestion
| Token for [API access](https://developers.qase.io/#authentication)                                       | `testops.api.token`        | `QASE_TESTOPS_API_TOKEN`        |                                         | Yes      | Any string                 |
```

Author: fixed

| Qase API host                                                                                                                                | `testops.api.host`         | `QASE_TESTOPS_API_HOST`         | `qase.io`                               | No       | Any string                 |

> | Code of your project (can be extracted from main page of your project: `https://app.qase.io/project/DEMOTR` - `DEMOTR` is project code here) | `testops.project`          | `QASE_TESTOPS_PROJECT`          |                                         | Yes      | Any string                 |

Reviewer: ```suggestion
| Code of your project, which you can take from the URL: `https://app.qase.io/project/DEMOTR` - `DEMOTR` is the project code | `testops.project`          | `QASE_TESTOPS_PROJECT`          |                                         | Yes      | Any string                 |
```
